canonical-websites/www.ubuntu.com
# Add docs for CDK 1.13 release

## Done

Added the following pages:
-  certs-and-trust.md
- docker-registry.md
- encryption-at-rest.md
- keepalived.md
- ldap.md
- using-vault.md

Updated these pages
- release-notes.md
- shared/_side-navigation.md
- troubleshooting.md



## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- View the docs at [http://0.0.0.0:8001/kubernetes/docs](http://0.0.0.0:8001/kubernetes/docs/)
